devicetypes/smartthings/zwave-battery-thermostat.src/zwave-battery-thermostat.groovy

```diff
@@ -150,7 +157,7 @@ def configure() {
 	16384	MECH STATUS
 	32768	SCP STATUS
 	*/
-	cmds << zwave.configurationV1.configurationSet(parameterNumber: 23, size: 2, scaledConfigurationValue: 8319).format()
+	cmds << zwave.configurationV1.configurationSet(parameterNumber: 23, size: 2, scaledConfigurationValue: 8319)
```

Comment:
I think this is the only one that needed to be fixed. Everything else wraps in a sendHubCommand call, which does take a list of ZWave commands

Reply:
You'll need to remove the `new physicalgraph.device.HubAction(` from each of those as well, since it doesn't have a `HubAction(physicalgraph.zwave.Command command)` constructor.

Reply:
right you are